wfe2/wfe_test.go

```diff
@@ -1931,6 +1983,31 @@ func TestGetCertificate(t *testing.T) {
 				test.AssertEquals(t, headers.Get(h), v)
 			}
 
+			if tc.ExpectedLinks != nil && len(tc.ExpectedLinks) > 0 {
+				links := headers["Link"]
```

Comment:
I think this can be significantly simplified. It looks like we are only ever defining a single expected link, so we can pare this down to something like:
```
hdr := `Link: <https://one.example.com>; rel="preconnect", <https://two.example.com>; rel="preconnect", <https://three.example.com>; rel="preconnect"`
expectedLink := `<https://two.example.com>; rel="preconnect"`
found := false
for _, link := range strings.Split(hdr, ", ") {
	if link == expectedLink { found = true; break }
}
```

If we wanted to use multiple expected links we'd then just need a single nested `for`, but since we aren't testing for multiple links, it makes sense to just use a `string` rather than `map[string][]string `.